Assets/Scripts/Network/RoomJoiner.cs

```diff
@@ -9,6 +10,7 @@ public class RoomJoiner : MonoBehaviourPunCallbacks
 {
     [SerializeField] private TMP_InputField roomNameInputField;
     [SerializeField] private string roomSceneName;
+    [SerializeField] private GameObject errorMessage;
```

Comment:
Perhaps change the type here from `GameObject` to `TextMeshProUGUI` so that you don't have to use `GetComponent` on the GameObject when setting the text (which can be costly). You can still drag the same object into this field in the inspector.

Reply:
Teach me senpai

Reply:
No 🗿